downstream/modules/platform/ref-upgrade-2.5-2.6-hub.adoc

```diff
@@ -0,0 +1,32 @@
+:_mod-docs-content-type: REFERENCE
+
+[id="ref-upgrade-2.5-2.6-hub"]
+
+= {HubNameStart}
+
+The following apply:
+
+* A {PrivateHubName} admin (Automation Content Administrator) in 2.5 will be removed in the upgraded version and for this user the permissions must be reassigned manually as part of the data movement process. 
+
+[IMPORTANT]
+====
+If teams with the same name exist in both {HubName} and within the platform-wide authentication gateway, users from {HubName} will be automatically added to corresponding teams within the platform-wide authentication gateway, and new teams will be created if they do not exist. 
+This approach aims to retain team memberships, but requires careful review of permissions post-upgrade.
+====
+
+* If you rely on {HubName} _Single Sign-On_ (SSO) to access the {HubName} user interface (UI), {HubName} SSO logins will no longer function after the upgrade. 
+However, API tokens will remain active. 
+Therefore, automated processes or systems that use API tokens for authentication will continue to operate without interruption. 
+If your workflows predominantly rely on API access, the impact may be minimal. 
+However, if users primarily access the UI through SSO, they will need to take action post-upgrade.
+
+* To restore UI access for users who previously relied on automation hub SSO, you need to reconfigure SSO within {PlatformNameShort} to be able to login. 
+For further information, see link:https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.4/html-single/installing_and_configuring_central_authentication_for_the_ansible_automation_platform/index#configuring-central-auth-generic-oidc-settings[Configuring Ansible Automation Platform Central Authentication Generic OIDC Settings and  Red Hat SSO/KEYCLOAK for Red Hat SSO and Ansible Automation Platform].
```

Comment:
Minor nit for title (can't recall if we're required to use attributes in the title for pre-migration work; if so, replace "Ansible Automation Platform" with the attribute: 
```suggestion
For further information, see link:https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.4/html-single/installing_and_configuring_central_authentication_for_the_ansible_automation_platform/index#configuring-central-auth-generic-oidc-settings[Configuring Ansible Automation Platform Central Authentication Generic OIDC Settings and  Red Hat SSO/keycloak for Red Hat SSO and Ansible Automation Platform].
```